src/main.py

```diff
@@ -1,8 +1,8 @@
-from gui import App
-from data.client_repository import ClientRepository
+from gui.__init__airline import App
+from data.airline_repository import AirlineRepository
 
-client_repo = ClientRepository("src/record/clients.jsonl")
+airline_repo = AirlineRepository("src/record/airlines.jsonl")
 
-app = App(client_repo)
+app = App(airline_repo)
```

Comment:
This is not correct, App should take both repositories as argument

Reply:
no problem.